Resilio Connect API/JSMonitor/data-store.js

```diff
@@ -40,3 +41,14 @@ function getJobProperty(id, propertyName) {
         throw "No such id or propertyName";
     }    
 }
+
+function enumerateAgents() {
+    let listAllAgents = []
+    for(var key in agents) {
+        if(agents.hasOwnProperty(key)) {
+            listAllAgents.push(Number(key));
```

Comment:
Is it needed to call Number()?

Reply:
Number was just to make sure the id came out as an integer not a string. Doesn't require the above if statement.